doc/docs/download_cfd.md

```diff
@@ -3,32 +3,40 @@
 Download nRF Connect for Desktop's executable from Nordic Semiconductor's [website](https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Connect-for-desktop).
 
 All nRF Connect for Desktop applications require the launcher v4.1.0 or later.
+See also the list of [Supported operating systems](./os_support.md).
 
 After installing and starting the launcher, you will see the list of available apps.
 See the [Launcher overview](overview_cfd.md) and [Installing nRF Connect for Desktop apps](installing_apps.md) pages for more information.
 
 ## Additional requirements
 
-Running nRF Connect for Desktop on macOS and Linux has additional requirements.
+Running nRF Connect for Desktop has the following additional requirements.
 
-### J-Link driver on macOS and Linux
+### SEGGER J-Link driver
 
-You must install SEGGER J-Link driver separately on macOS and Linux. To do so, download and
-install it from [SEGGER](https://www.segger.com/downloads/jlink) under the
-section _J-Link Software and Documentation Pack_.
+Required on all platforms. On Windows, the driver comes bundled with nRF Connect for Desktop.
 
-If you don't install the J-Link driver, running the apps will fail and you will get error messages
-`CouldNotFindJprogDLL`, `CouldNotOpenDLL`, `JLINKARM_DLL_NOT_FOUND`, or similar.
+On macOS and Linux, you must install the driver manually.
+Download the installer for your platform from [SEGGER J-Link Software](https://www.segger.com/downloads/jlink/#J-LinkSoftwareAndDocumentationPack).
+If you don't install the J-Link driver, running the apps will fail and you will get error messages `CouldNotFindJprogDLL`, `CouldNotOpenDLL`, `JLINKARM_DLL_NOT_FOUND`, or similar.
```

Comment:
Running of the apps won't fail, but usage will be highly limited.
For example Toolchain manager can be used without JLink at all as there are no devices connected